### ISSUE TITLE
ogrinfo --formats: also show filename extensions

### ISSUE THREAD

**Issue by jidanni** [OSGeo/gdal#8958]
```
$ ogrinfo --formats
Supported Formats:
  FITS -raster,vector- (rw+): Flexible Image Transport System
  PCIDSK -raster,vector- (rw+v): PCIDSK Database File
  netCDF -raster,multidimensional raster,vector- (rw+s): Network Common Data Format...
```
That's great, but perhaps at the end of each line add the file
extensions associated, yes even if not exclusively, for that
format/driver. Perhaps enclose them in parentheses.

I mean there is some table of these the program uses. So this would be
a great place to show it.

And that way a user who thinks, "Today I shall produce my first SQLite
file. Hmmm, how shall I name it?" Well he could see in the list
".sql". So then would know "Ah, I'll call my output file "myfile.sql"!

And then he wouldn't even need ogr2ogr's "-f".

See also
https://gis.stackexchange.com/questions/175610/list-of-gdal-raster-file-extensions

**Comment by rouault** [comment]
That already exists:
```
$ ogrinfo --format sqlite
Format Details:
  Short Name: SQLite
  Long Name: SQLite / Spatialite / RasterLite2
  Supports: Raster
  Supports: Vector
  Extensions: sqlite db
  Help Topic: drivers/vector/sqlite.html
[... snip ...]
```

**Comment by jidanni** [comment]
That's great, except the user needs to look at each one to get the info one by one.

Well yes, --formats can only be so wide.

(Anyway the other day I did
```
ogrinfo --formats|perl -anwle 'next unless /vector.*\(rw/; print $F[0];' > formats
for i in $(cat formats); do ogr2ogr -f $i /tmp/f.$i -limit 1 \
-sql "SELECT FRSTDIVID FROM PLSSFirstDivision"; done
```
to see what each format looked like.

As you see I ended up using the long name for the file extension, because... well even now that I know where they are listed, I would need to add complexity to my script to access them.)
